src/constants.ts

```diff
@@ -62,3 +62,80 @@ export const RPC_URLS = {
     websocket: 'wss://wss.calibration.node.glif.io/apigw/lotus/rpc/v1'
   }
 } as const
+
+/**
+ * PDP service contract addresses (proxy addresses for upgradeable contracts)
+ */
+export const PDP_SERVICE_CONTRACT_ADDRESSES = {
+  mainnet: '', // TODO: Get actual mainnet address from deployment
+  calibration: '0x2B76E983d30553E7717547230670D4F4F4d813aC'
+} as const
+
+/**
+ * PDP Verifier contract addresses
+ */
+export const PDP_VERIFIER_ADDRESSES = {
+  mainnet: '', // TODO: Get actual mainnet address from deployment
```

Comment:
`0x9C65E8E57C98cCc040A3d825556832EA1e9f4Df6` 
You can find this value in the filozone/pdp README.md